app/controllers/pages_controller.rb

```diff
@@ -27,6 +27,6 @@ def privacy
     def marketing_site?
       return true if RadConfig.force_marketing_site?
 
-      RadConfig.allow_marketing_site? && request.host.start_with?('www.')
+      RadConfig.allow_marketing_site? && request.host.start_with?("#{RadConfig.marketing_subdomain}.")
```

Comment:
we're using wwwtest for the PUG for now, could probably remove this config setting once we switch that to www